app/views/spaces/show.html.erb

```diff
@@ -2,14 +2,13 @@
 <p>Find me in app/views/spaces/show.html.erb</p>
 
 <ul class="list-group">
-  <% spaces.each do |space| %>
     <li class="list-group-item">
-      <%= space.location %>
-      <%= space.description %>
-      <%= space.volume %>
-      <%= space.price_per_month %>
+      <%= @space.location %>
+      <%= @space.description %>
+      <%= @space.volume %>
+      <%= @space.price_per_month %>
       </li>
   <% end %>
 
-<%= link_to "book this space", new_space_reservation_path(@space) %>
-<%= link_to "back to all spaces", spaces_path %>
+<%= link_to "BOOK THIS SPACE", new_space_reservation_path(@space) %>
+<%= link_to "VIEW ALL SPACES", spaces_path %>
```

Comment:
All caps works, too ... just gotta keep it in mind to ensure consistency across pages 👍